public/locales/fr/common.json

```diff
@@ -182,6 +185,7 @@
   "confirmationP2": "Nous vous inviterons à prendre part aux expériences et à d’autres activités de recherche au fur et à mesure qu’elles se présenteront afin d’améliorer Service Canada pour tous.",
   "confirmationP3": "N’oubliez pas que vous pouvez choisir le temps que vous y consacrez et que chaque petit geste compte.",
   "confirmationP4": "Nous sommes véritablement heureux de vous avoir dans l’équipe!",
+  "resendEmail": "Renvoyer le courriel",
```

Comment:
The word doc with initial translations has this as `Envoyer de nouveau un courriel de vérification `